commit fd98a28d6e94080e52bbedc789b06606a6019b10
Author: Lukas Slebodnik <lslebodn@redhat.com>
Date:   Wed Jun 12 13:24:12 2013 +0200

    Change order of libraries in linking process.
    
    It seems that some linkers have problem with wrong order of libraries.
    This commit only change order.

diff --git a/Makefile.am b/Makefile.am
index 93e3a6f..88e29ff 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -577,10 +577,10 @@ endif
 libsss_util_la_LDFLAGS = -avoid-version
 
 SSSD_INTERNAL_LTLIBS = \
+    libsss_util.la \
     libsss_crypt.la \
     libsss_debug.la \
-    libsss_child.la \
-    libsss_util.la
+    libsss_child.la
 
 lib_LTLIBRARIES = libipa_hbac.la libsss_idmap.la libsss_nss_idmap.la
 dist_pkgconfig_DATA += src/providers/ipa/ipa_hbac.pc
